### ISSUE TITLE
[Feature Request] Dynamically-Compiled Payloads

### ISSUE THREAD

**Issue by w0rk3r** [preludeorg/operator-support#450]
It would be nice to compile at least Csharp code as it is done on the Caldera Builder plugin. Currently, distributing custom binaries across the team, uploading them, debugging, and adding functionality requires way more effort than it could if we had similar functionality.

https://caldera.readthedocs.io/en/3.1.0/Dynamically-Compiled-Payloads.html

**Comment by privateducky** [comment]
Yep, definitely on our list! 

Lucky for us, @khyberspache wrote that builder plugin ;) and has some ideas for how to do a version of it for Operator. 

We've been using a dynamic payload compiling feature on our backend, to supply the Operator payloads we create. But we need to move that behind an API or similar to allow others to start using it as well. 

More to come on this topic.

**Comment by w0rk3r** [comment]
Yeah, I think I talked with Alex about this last year, but now it is definitely on my wishlist, hehe

Looking forward to it!

**Comment by privateducky** [comment]
As a predication; I'd look for this in the v1.7-v1.8 time period.

**Comment by privateducky** [comment]
Update: 1.5 was just released yesterday. this ticket was not in that release (as predicted) but know it's still on our radar.
